src/ddl/migrate_21.1_to_22.1.sql

```diff
@@ -1,8 +1,8 @@
 /*
-  Migration Script for Release 21.1 to 21.2
+  Migration Script for Release 21.1 to 22.1
 */
 
-PROMPT >> Running DDL for Upgrade from 21.1 to 21.2
+PROMPT >> Running DDL for Upgrade from 21.1 to 22.1
 PROMPT >> -------------------------------------------
 
 PROMPT >> Remove obsolete objects
```

Comment:
Removed Page APIs need to be included here with drop commands.

Reply:
Done in https://github.com/mt-ag/apex-flowsforapex/pull/396/commits/73956c03478c00f907b20eeda9a2e20a475fb911